<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shrishti_BT" failures="1" tests="3" name="CheckoutTest" time="30.782" errors="0" timestamp="2023-06-07T13:20:48 IST">
  <testcase classname="ECommerce.tests.CheckoutTest" name="EcommerceE2E" time="12.079">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at ECommerce.tests.CheckoutTest.EcommerceE2E(CheckoutTest.java:71)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- EcommerceE2E -->
  <testcase classname="ECommerce.tests.CheckoutTest" name="EcommerceE2E" time="9.213"/>
  <testcase classname="ECommerce.tests.CheckoutTest" name="OrderHistory" time="0.0">
    <skipped/>
  </testcase> <!-- OrderHistory -->
</testsuite> <!-- CheckoutTest -->
